projects/rich-previews/src/google-product-mobile/google-product-mobile.js

```diff
@@ -0,0 +1,48 @@
+const app = Vue.createApp({
+  setup() {
+    const params = new URL(window.location.href).searchParams;
```

Comment:
Shouldn't these `const`'s be a `ref` or `computed` property?

Reply:
Nee dat zijn ze ook niet op de andere previews, deze data verandert niet.